internal/bgp/native/native.go

```diff
@@ -64,19 +65,20 @@ func NewSessionManager() *sessionManager {
 //
 // The session will immediately try to connect and synchronize its
 // local state with the peer.
-func (sm *sessionManager) NewSession(l log.Logger, addr string, srcAddr net.IP, myASN uint32, routerID net.IP, asn uint32, holdTime time.Duration, password string, myNode string) (bgp.Session, error) {
+func (sm *sessionManager) NewSession(l log.Logger, addr string, srcAddr net.IP, myASN uint32, routerID net.IP, asn uint32, holdTime time.Duration, keepaliveTime time.Duration, password string, myNode string) (bgp.Session, error) {
 	ret := &session{
-		addr:        addr,
-		srcAddr:     srcAddr,
-		myASN:       myASN,
-		routerID:    routerID.To4(),
-		myNode:      myNode,
-		asn:         asn,
-		holdTime:    holdTime,
-		logger:      log.With(l, "peer", addr, "localASN", myASN, "peerASN", asn),
-		newHoldTime: make(chan bool, 1),
-		advertised:  map[string]*bgp.Advertisement{},
-		password:    password,
+		addr:          addr,
+		srcAddr:       srcAddr,
+		myASN:         myASN,
+		routerID:      routerID.To4(),
+		myNode:        myNode,
+		asn:           asn,
+		holdTime:      holdTime,
+		keepaliveTime: keepaliveTime,
+		logger:        log.With(l, "peer", addr, "localASN", myASN, "peerASN", asn),
+		newHoldTime:   make(chan bool, 1),
```

Comment:
If the value is not supported with the native stack, we should fail in parsing the configuration.

Reply:
ok